src/Pages/OtherProjects/OtherProjects.css

```diff
@@ -30,3 +30,50 @@
     object-fit: none;
     height: 300px;
 }
+
+@media only screen and (max-width: 1100px) {
+    .otherProjectsIntroContainer {
+        align-items: center;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+}
+
+@media only screen and (max-width: 700px) {
```

Comment:
I feel like you don't need both the 700px and 600px media queries since they apply exactly the same styling